packages/replay-next/components/sources/log-point-panel/LogPointPanel.tsx

```diff
@@ -84,7 +84,7 @@ function PointPanel(props: ExternalProps) {
   const [hitPoints, hitPointStatus] = getHitPointsForLocationSuspense(
     client,
     point.location,
-    null,
+    point.condition,
```

Comment:
Aw, hell

Reply:
😂

Reply:
Didn't think about this at the time, but this change means that editing a point's condition text (in the point panel overlay) will cause the point panel to re-suspend.

(Looking into this as part of FE-1138)